docs/hardware/pcb_design.md

```diff
@@ -0,0 +1,76 @@
+2
+module also has an EEPROM chip named AT24C32, this chip will help save all
+metering information such as STS token, tariff and much more as needed. 
+
+#### **3. 16x2 LCD**
+
+A 16x2 LCD from Hitachi is proposed and used in this design to display all necessaryparameters such as RMS voltage, current, power, energy, Top-up balance and visuallysee STS token number inputted for manual Top-up in event of network failure, thisLCD serve as physical interactive medium between electrical parameter measuredanduser readable and understandable information. 
+
+#### **4. Power board design**
+
+The power board design for the energy meter is a SMPS (Switch Mode Power Supply)
+based design with a fly back forward switching topology using UC3843 current
+controller IC and ferrite core transformer, this topology was selected as opposedtousing transformer based design due to its reliability, weight, cost, sophisticationandperformance under low voltage/high voltage conditions without failure, also voltageregulator IC(Integrated Circuit) were used to regulate voltage as needed for the GSMmodule, micro-controller and other module needing regulated voltage, voltageregulator IC such as (L7805, LM350) were used in regulation, data-sheet to each ICisavailable on the data-sheet section of the deliverable set1.
+
+#### **5. Variable resistor calibration for VRms**
+
+ Variable resistor is incorporated into this design to enable calibration the ACvoltageas required. 
+
+#### **6. Electrically isolated hall effect sensor**
+
+The current sensing method used here had to be changed from hall effect current
+sensor to current transformer measuring method, this method is used to increaseefficiency. An accuracy test between hall effect current sensor and current transformer
+was performed and it was discovered that current transformer provides better
+accuracy and stability when measuring AC current as compared to hall effect current
+sensor. In addition, hall effect current sensors accuracy is affected by heat and moresusceptible to electromagnetic interference due to the sensors electromagnetic diesensing method used as proposed by **“Edwin Hall”**. 
+
+#### **7. Electromagnetic relay to turn users on/off**
+
+Magnetic latching relay is used in place of electromagnetic relay due to cost,
+3
+reliability, power consumption, size. Comparison table is highlighted below. 
+
+| Electromagnetic relay |  &nbsp; &nbsp; &nbsp;  &nbsp; &nbsp; &nbsp;   &nbsp; &nbsp; &nbsp;Magnetic latching relay|
```

Comment:
Can you remove all the `&nbsp;` from the docs? I don't think they are needed and just make the docs hard to read.